job-manager/jobstore-liquibase/src/main/resources/liquibase/jobstore-db-changelog.xml

```diff
@@ -7,5 +7,6 @@
     <include file="liquibase/jobstore-changesets/002a-add-job-data-column-h2.xml"/>
     <include file="liquibase/jobstore-changesets/002b-add-job-data-column-postgres.xml"/>
     <include file="liquibase/jobstore-changesets/003-add-retry-attempts-remaining-column.xml"/>
+    <include file="liquibase/jobstore-changesets/004-copy-retry-attempts-to-new-column.xml"/>
 
 </databaseChangeLog>
```

Comment:
It's annoying to have to maintain a list of all the files. Maybe it would be better to change this to

    <includeAll path="liquibase/jobstore-changesets"/>
    
(or whatever the path is)